docs/fides/docs/development/releases.md

```diff
@@ -21,3 +21,17 @@ Hotfixes are an exception to this and can be added and pushed as patch versions
 ## Branching
 
 Fidesctl uses continuous delivery with a single `main` branch. All code changes get merged into this branch. For releases, a new tag is created and the release process kicks off automatically. In the case of patches, a branch is created from the relevant tag and commits are then cherry-picked into it and a new patch version tag is created.
+
+## Release Steps
+
+We use GitHub’s `release` feature to tag releases that then get automatically deployed to DockerHub & PyPi via GitHub Actions pipelines. We also use a `CHANGELOG.md` to make sure that our users are never surprised about an upcoming change and can plan upgrades accordingly. The release steps are as follows:
+
+1. Open a PR that is titled the version of the release (i.e. `1.6.0`)
+    * Rename the `Unreleased` section of `CHANGELOG.md` to the new version number and put a date next to it
+    * Update the `compare` links for both the new version and for the new `Unreleased` section
+1. Once approved, merge the PR
```

Comment:
This feels clunky. Would it be OK to instead have the first PR opened as part of a new release add the new version number instead of `Unreleased`? Then we don't need to create an "update `CHANGELOG.md`" PR for every release. The new link would be broken until the release tag gets created, but users would be able to easily cross-reference the changelog with the releases tab to see that the latest version hasn't been released yet.

Reply:
Then we'd still have a step here at the end that says "after release, open a PR and add the new version". Either way we're adding a new PR, and I like the explicit `unreleased` section so people know what's in main but hasn't been released as opposed to needing to check our releases. Not to mention that breaks the convention of "keep a changelog"